server.js

```diff
@@ -0,0 +1,22 @@
+import express from "express";
+import bodyParser from "body-parser";
+import fs from "fs";
+
+const app = express();
+
+app.use(express.static("./public"))
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.get("/retrieve-last-answer", (_, res) => {
+	const lastAnswer = fs.readFileSync("./last-answer.dat");
+	res.send(lastAnswer);
+});
```

Comment:
An endpoint to read from our `last-answer.dat` file and respond to the request with the data.